packages/paste-website/src/pages/icon-system/how-to-add-an-icon.mdx

```diff
@@ -107,6 +120,8 @@ Written for Sketch 59.1
    2. You shouldn’t see `transform`. If you do, flatten your shape.
    3. You should see `width=”24" height="24"`. If you don’t, check that the artboard you’re
       exporting is 24px × 24px and you’re exporting the artboard, not the layer.
+   4. If you're adding an icon used to brand a product, add the word "Product" to the 
```

Comment:
My instructions here weren't very good lol.

Can you add this as step 4 to the parent list, i.e., "Commit your changes..." would become step 5.